ccm_web/package.json

```diff
@@ -10,7 +10,6 @@
     "build:client": "webpack --config webpack/webpack.prod.ts",
     "build:server": "tsc --project tsconfig.build.json",
     "preprod": "npm run build:client && npm run build:server",
```

Comment:
We could change the name of the `preprod` command, maybe just to `build`. The `pre*` construction makes less sense if we don't have a `prod` (it already didn't make too much sense, since we don't use `prod` in the Docker process).

Reply:
Done [`bfa4929` (#125)](https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/125/commits/bfa4929fc93af2504b3d05abe82d631dc6a602f4)